### ISSUE TITLE
Feature Request: Full 60FPS HDMI output @ 1080p 

### ISSUE THREAD

**Issue by keenan-smith** [gopro/labs#180]
Not sure if you guys over at labs can make this possible, but it's been an issue that has plagued GoPro for a few generations now.

The GoPro specs state that it outputs 1080p60 over HDMI, but this is actually false - it's only 30fps with every other frame copied.
https://community.gopro.com/s/article/Media-Mod-HDMI-Output?language=en_US

On the Hero 9 you could fix this by recording while outputting to HDMI (this would prevent the copied frames and output true 60fps), however recording while outputting to HDMI on the 10 and up now forces the HDMI resolution to 480p.

TL;DR: GoPro says that they should output 1080p60 at idle but it's really 1080p30

**Comment by Hunter-belaxbox** [comment]
Please fix! Many of us want to use our GoPros for live HDMI output at 1080 60fps.

**Comment by dnewman-gpsw** [comment]
I just tested HERO11 and the output is 1080p60 when monitoring, not recording, just like the documentation says.  I was connected to an LG 1080p display/TV, and recording the display with another GoPro to confirm its output. A tad crude, but I confirmed there was no frame duplications.  The issue you might see on older or cheaper displays or HDMI equipment, is they might not support 1080p60, but rather 1080i60.  In this case the GoPro, which is not interlaced, will output 30p at 60i.  

One benefit from this investigation, the next HERO11 Lab release will support QR code reading while connected to HDMI.

**Comment by Hunter-belaxbox** [comment]
Thanks for testing!

**Comment by Marlow925** [comment]
> I just tested HERO11 and the output is 1080p60 when monitoring, not recording, just like the documentation says. I was connected to an LG 1080p display/TV, and recording the display with another GoPro to confirm its output.

Have you been frame peeping on that ? With something that updates more often than 60 fps ? Because it's not been the case form a long time. Certainly since the Hero 5 Black. 

Additionally GoPro completely screwed the HDMI output with recording since the Hero 10. I reported the bug to them in November 2021, but have been ignored by support since the start of 2022, while the bug still is supposed to be open. 

Bug reference is: ref:_00Do0HJuF._5003b1dVSHM:ref

Recording while grabbing HDMI used to be the fix for the GoPro Hero 8 and Hero 9 to get true 1080p 60fps. As of the GoPro Hero 10 Black, the camera falls back to 480p, when recording in 16:9 1080p60, which is completely contradicting GoPro's own documentation, that can be found here:  https://community.gopro.com/s/article/Media-Mod-HDMI-Output?language=en_US

/M

**Comment by keenan-smith** [comment]
Just tested HDMI output on the Hero 10 with an Elgato Camlink 4K, recorded with OBS - seems that it is outputting true 60FPS@1080p, no duplicated frames. And it looks like the output table on the GoPro website has been changed to reflect that the Hero 10 only outputs 480p while recording.

**Comment by Marlow925** [comment]
> Just tested HDMI output on the Hero 10 with an Elgato Camlink 4K, recorded with OBS - seems that it is outputting true 60FPS@1080p, no duplicated frames. And it looks like the output table on the GoPro website has been changed to reflect that the Hero 10 only outputs 480p while recording.

Changing the table is not fixing the problem though. Changing the table is moving the goal posts.

/M

**Comment by dnewman-gpsw** [comment]
From the question in the original post, the HERO11 and HERO10 are working correctly.
